easybuild/easyconfigs/r/Reapr/Reapr-1.0.18-foss-2019a-Perl-5.28.1.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'MakeCp'
+
+name = 'Reapr'
+version = '1.0.18'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://www.sanger.ac.uk/science/tools/reapr'
+description = """A tool that evaluates the accuracy of a genome assembly using mapped paired end reads, 
+without the use of a reference genome for comparison."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['ftp://ftp.sanger.ac.uk/pub/resources/software/reapr/']
+sources = ['Reapr_%(version)s.tar.gz']
+patches = ['Reapr-1.0.18-cleanup.patch']
+checksums = [
+    '6d691b5b49c58aef332e771d339e32097a7696e9c68bd8f16808b46d648b6660',  # Reapr_1.0.18.tar.gz
+    '87131b45635815971e6e583712008043c3d077205127b50fbb09b9b5289b3b20'   # Reapr-1.0.18-cleanup.patch
+]
+
+dependencies = [
+    ('Perl', '5.28.1'),
+    ('R', '3.6.0'),
+    ('BamTools', '2.5.1'),
+    ('SAMtools', '1.9'),
+]
+
+builddependencies = [('CMake', '3.13.3')]
+
+exts_defaultclass = 'PerlModule'
+
+exts_list = [
+    ('File::Spec::Link', '0.140', {
```

Comment:
Maybe we should add this to the `Perl` easyconfig instead?

Reply:
I agree.

Reply:
BTW. I had to tell another user that needed "Palavras", how she could install the Perl libraries "DB_File" and "MLDBM::Sync" in her home-dir.